server/src/test/java/com/objectcomputing/checkins/endtoend/pages/HomePage.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * 
+ */
+package com.objectcomputing.checkins.endtoend.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author tajr
+ * 
+ * This class will store all the locators and methods of home page.
+ *
+ */
+public class HomePage {
+	
+	WebDriver driver;
+	
+	public HomePage(WebDriver ldriver)
+	{
+		this.driver = ldriver;
+	}
+	
+	@FindBy(xpath="/html[1]/body[1]/div[1]/form[1]/fieldset[1]/label[1]") 
+	WebElement chooseafile_btn;
+	
+	@FindBy(id="file")
+	WebElement filepath;
+	
+	@FindBy(name="submit")
+	WebElement upload_btn;
+
+	@FindBy(className="success")
+	WebElement success_msg;
+	
+	@FindBy(className="error")
+	WebElement error_msg;
+	
+	
+	public void uploadFile() throws InterruptedException
+	{
+		// TODO needs to be fixed
+		filepath.sendKeys("/Users/tajr/Desktop/File.rtf");
```

Comment:
Local file we'll have to deal with later.

Reply:
@hwilliams314 would you mind changing this file to be something local to the project. pick something simple like the application.yml. It really doesn't matter what it is, so long as it's small.